designs/administrator/future-openshift/autoscaler-guidance/index.md

```diff
@@ -0,0 +1,63 @@
+---
+parent: Administrator
+version: future-openshift
+---
+
+# Machine and Cluster Autoscaler guidance
+
+The flow below helps users out by pointing them toward configuring the Cluster Autoscaler if they haven’t already done so by the time they try to create a Machine Autoscaler. It also shows a user whether a MachineSet has a Machine Autoscaler related to it in the MachineSet list.
+
+## Creating and Editing Cluster Autoscaler
+
+Users will likely first encounter OpenShift’s autoscaling feature from within the Machine Autoscalers nav item.
+
+The user clicks “Create Machine Autoscaler” and is directed to the screen below, if a Cluster Autoscaler has not already been configured:
+
+![Autoscaler](img/autoscaler-0.png)
+
+A brief description of what autoscaling is and how Machine Autoscalers are related to the Cluster Autoscaler is included at the top of the creation form, with a link to the current documentation for users who’d like that.
+
+If the user hasn’t already configured the Cluster Autoscaler (which is likely), then an inline info alert prominently suggests that they do so first before continuing.
+
+The user clicks “Create Cluster Autoscaler.” and is directed to the following form
+![Autoscaler](img/autoscaler-1.png)
+
+The Cluster Autoscaler creation form is largely filled out with defaults aside from the GPU count section, which are unchecked by default. The Maximum number of nodes should be the current number of nodes at the time that this page is loaded.
+
+The maximum thresholds of the “Scaling limits” section are based on the total CPU core and Memory counts of all of the non-master nodes in the cluster at the time that the creation page is loaded.
+
+The user clicks “Create”
+![Autoscaler](img/autoscaler-2.png)
```

Comment:
Could the user be able to remove the cluster autoscaler right from this page? Maybe another link like "Delete Autoscaler"? Maybe it is advantageous to have them look at the autoscaler in the Edit form before they delete it though.

Reply:
Can we update this page or crop the top off? Cluster settings was reworked in 4.6. We may want to have the pencil here to edit. I don't know if there's any other details pages we explicitly say edit in a link like that.

Reply:
@itsptk Agreed, it's a bit of a weird case. I remember I tried adding "Delete Autoscaler" in red and it felt kinda floaty and too prominent. I somewhat wonder if a single-row table could be added to show the current Cluster Autoscaler configuration and then we'd have a kebab menu to Edit/Delete as usual, but maybe that would be too heavy for this one feature?

Reply:
@beanh66 Cropping/blurring the top part that's in flux makes sense. 👍

@LPutney The [Editing convention documentation](http://openshift.github.io/openshift-origin-design/conventions/documentation/edit.html#edit-links) has examples of how pencils are used. I think tweaking this a bit to have "Enabled" be blue text with a pencil next to it would be aligned.

@beanh66 FWIW some Details pages now have `Edit` links with a pencil which @itsptk captured in the [Editing convention](http://openshift.github.io/openshift-origin-design/conventions/documentation/edit.html#edit-links), but that's for editable groups of stuff which this isn't.

![image](https://user-images.githubusercontent.com/9122899/90800334-84fed180-e2e2-11ea-87c1-53c0f895873e.png)